src/Microsoft.Graph.Core/Exceptions/ServiceException.cs

```diff
@@ -37,15 +37,5 @@ public bool IsMatch(string errorCode)
 
             return false;
         }
-
-        public override string ToString()
```

Comment:
First off, thank you for finding this. I see why it should be removed. I'm unsure how it would be used.. I'm just concerned in case customers have taken a dependency on this bad behavior.  I'm leaning towards removing it as you have.